pkg/controller/autoimport/autoimport_controller.go

```diff
@@ -110,37 +109,6 @@ func (r *ReconcileAutoImport) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, err
 	}
 
-	lastRetry := 0
-	totalRetry := 1
-	if len(autoImportSecret.Annotations[constants.AnnotationAutoImportCurrentRetry]) != 0 {
-		lastRetry, err = strconv.Atoi(autoImportSecret.Annotations[constants.AnnotationAutoImportCurrentRetry])
-		if err != nil {
-			return reconcile.Result{}, err
-		}
-	}
-
-	if len(autoImportSecret.Data[constants.AutoImportRetryName]) != 0 {
-		totalRetry, err = strconv.Atoi(string(autoImportSecret.Data[constants.AutoImportRetryName]))
```

Comment:
After the AutoImportRetryName is ignored, how to determinal if a cluster is imported failed from the web UI?

Reply:
It can be determined by checking the `reason' of the condition `ManagedClusterImportSucceeded` is `ConditionReasonManagedClusterImportFailed` or not.